diff --git a/node_modules/semantic-release/lib/definitions/constants.js b/node_modules/semantic-release/lib/definitions/constants.js
index 83bb24b..10477cc 100644
--- a/node_modules/semantic-release/lib/definitions/constants.js
+++ b/node_modules/semantic-release/lib/definitions/constants.js
@@ -1,6 +1,6 @@
 export const RELEASE_TYPE = ["patch", "minor", "major"];
 
-export const FIRST_RELEASE = "1.0.0";
+export const FIRST_RELEASE = "1.0.1";
 
 export const FIRSTPRERELEASE = "1";
 
